styles/style.css

```diff
@@ -0,0 +1,358 @@
+* {
+  box-sizing: border-box;
+  margin: auto;
+}
+main {
+  padding: 0.5rem 9.5rem 4rem 9.5rem;
+}
+img {
+  max-width: 100%;
+}
+ul {
+  list-style: none;
+}
+a:link {
+  text-decoration: none;
+}
+/* header style start */
+header > div {
+  background-image: url('../images/header-image.jpg');
+  display: grid;
+  padding-left: 5rem;
+  padding-right: 6rem;
+  padding-top: 2rem;
+  padding-bottom: 6rem;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-template-columns: 36% 1fr 30%;
+}
+header > div * {
+  margin: 0;
+}
+header img[alt="a white dog logo image for header"] {
+
+}
+header img[alt="a white color cart icon in the header"] {
```

Comment:
not sure what this is, but don't put alt tags in your CSS, they always belong in the HTML